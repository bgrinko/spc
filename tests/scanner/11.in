m_123_2323_lsw