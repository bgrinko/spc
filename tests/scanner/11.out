1	14	IDENTIF		m_123_2323_lsw
